engine/src/main/resources/templates/ResourceOptimizationService/dailyEmailBody.html

```diff
@@ -0,0 +1,26 @@
+{#include ResourceOptimizationService/insightsEmailBody}
+{#content-title}Daily Resource Optimization Service Summary - {action.context.start_time.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <p>Resource Optimization Service has identified <b> {action.context.system_count}</b> system{#if action.context.system_count > 1}s{/if} with new suggestion{#if action.context.system_count > 1}s{/if}.
```

Comment:
Since one system can have multiple suggestions, using a plural all the time from suggestions would make sense here:
```suggestion
        <p>Resource Optimization Service has identified <b> {action.context.system_count}</b> system{#if action.context.system_count > 1}s{/if} with new suggestions{/if}.
```